pkg/apis/jenkins/v1alpha2/jenkins_types.go

```diff
@@ -310,6 +310,9 @@ type JenkinsMaster struct {
 	// Plugins contains plugins required by user
 	// +optional
 	Plugins []Plugin `json:"plugins,omitempty"`
+
+	// DisableCSRFProtection allows you to toggle CSRF Protection on Jenkins
+	DisableCSRFProtection bool `json:"disableCSRFProtection,omitempty"`
```

Comment:
Please remove `omitempty` to always show it in CR.

Reply:
I have removed omitempty.